src/commands/GameIntegration/ffxiv.ts

```diff
@@ -81,10 +82,10 @@ export default class extends RichDisplayCommand {
 		const titles = language.get(LanguageKeys.Commands.GameIntegration.FFXIVCharacterFields);
 
 		const display = new UserRichDisplay(
-			new MessageEmbed()
+			new Embed()
```

Comment:
This doesn't use `.splitFields` so it can just be `MessageEmbed`.

Reply:
Resolved in 9cbfa8fc.